File: Server/src/main/java/co/kr/dgsw/searchvoca/domain/Vocabulary.java
package co.kr.dgsw.searchvoca.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Vocabulary {
    private int id;

    private String name;

    private String explanation;

    private String language;
}

File: Android/domain/src/main/java/co/kr/searchvoca/domain/model/Vocabulary.kt
package co.kr.searchvoca.domain.model

data class Vocabulary(
    val id: Int? = null,
    val name: String,
    val explanation: String? = null
//    val language: String,
//    val isCorrections: Boolean = false,
//    val date: Long? = null
)

// todo 다른 방법
val default = Vocabulary(name = "미정")